docs/quick.rst

```diff
@@ -53,19 +53,25 @@ Just the code
     >>> import pandas as pd
     >>> import nannyml as nml
     >>> from IPython.display import display
+
+    >>> # Load dummy data
```

Comment:
Maybe not dummy data but synthetic data?

Reply:
Done